docs/api_wiki/sonichost_methods/get_interfaces_status.md

```diff
@@ -0,0 +1,56 @@
+# get_interfaces_status
```

Comment:
@Blueve I think this doc also need your review, thanks!

Reply:
lgtm